wazimap_ng/profile/serializers/indicator_data_summary_serializer.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+from django.db.models import F
+
+from wazimap_ng.utils import mergedict, qsdict
+from wazimap_ng.datasets.models import IndicatorData
+
+
+logger = logging.getLogger(__name__)
+
+def IndicatorDataSummarySerializer(profile, geography, version):
+    child_geographies = geography.get_child_geographies(version)
+    children_indicator_data = (IndicatorData.objects
+            .filter(
+                indicator__in=profile.indicators.all(),
+                indicator__profileindicator__profile=profile,
+                geography__in=child_geographies,
+                indicator__dataset__version=version,
+            )
+            .values(
+                profile_indicator_id=F("indicator__profileindicator__id"),
+                profile_indicator_label=F("indicator__profileindicator__label"),
+                subcategory=F("indicator__profileindicator__subcategory__name"),
+                subcategory_id=F("indicator__profileindicator__subcategory_id"),
+                category=F("indicator__profileindicator__subcategory__category__name"),
+            )
+            .order_by("indicator__profileindicator__order")
+            )
+
+    d5 = qsdict(children_indicator_data,
+        "category",
+        lambda x: "subcategories",
+        "subcategory",
+        lambda x: "indicators",
+        lambda x: x["profile_indicator_label"],
+        lambda x: "id",
+        "profile_indicator_id",
+    )
+
+    new_dict = {}
+    mergedict(new_dict, d5)
+    return new_dict
```

Comment:
Why does this merge with an empty dictionary. Can we not export just d5
Also d5 could be more descriptive like indicator_summary

Reply:
updated this code ..
now we have multiple data dicts